src/js/models/color.js

```diff
@@ -96,9 +96,9 @@ define(function (require, exports, module) {
         }
 
         return new Color({
-            "r": rgb.red,
-            "g": green,
-            "b": rgb.blue,
+            "r": mathjs.round(rgb.red),
```

Comment:
FYI: This is because colors for type layers were returning as floating points and vector layers colors were integers. Created a problem when we called .uniformValue() on a group of type and vector colors, even though they were the same.